NN_2L2H2O_poly2d/NN_2L2H2O_poly2d.in

```diff
@@ -0,0 +1,25 @@
+float X[][69] = { 
+{   0.63555193  ,  0.28013600  ,  0.07094245  ,  0.28219061  ,  0.31406159  ,  0.27602624  ,  0.03477652  ,  0.10947368  ,  0.13645770  ,  0.04520253  ,  0.11852976  ,  0.08946130  ,  0.20893379  ,  0.08953063  ,  0.09473357  ,  0.08797996  ,  0.42408922  ,  0.07732489  ,  0.02228030  ,  0.00908176  ,  0.02624278  ,  0.21359981  ,  0.05401579  ,  0.48107120  ,  0.11886203  ,  0.24053560  ,  0.03267345  ,  0.00971775  ,  0.07789201  ,  0.00592623  ,  0.04441366  ,  0.03046374  ,  0.03674165  ,  0.02001929  ,  0.02000488  ,  0.13645770  ,  0.06595533  ,  0.08851034  ,  0.04468295  ,  0.01990608  ,  0.12029455  ,  0.24053560  ,  0.02324058  ,  0.23772406  ,  0.01904762  ,  0.01868030  ,  0.01958198  ,  0.11886203  ,  0.03971281  ,  0.06743135  ,  0.07057794  ,  0.04421157  ,  0.04508761  ,  0.06901279  ,  0.06058841  ,  0.20893379  ,  0.09384801  ,  0.03933894  ,  0.21359981  ,  0.01987353  ,  0.01980258  ,  0.06058841  ,  0.10739758  ,  0.06743135  ,  0.01961904  ,  0.03398999  ,  0.00990724  ,  0.01848377  ,  0.00503283  },
```

Comment:
This is only for testing against the python implementation so may be overkill. In the implementation we will compute the feature vector for the NN input from Cartesian coordinates of atoms in the system. This will be the next step of the implementation: Behler-Parinello symmetry functions that compute NN input for each atom (we drop polynomials as features).